lib/ae_page_objects/elements/collection.rb

```diff
@@ -41,7 +41,7 @@ def each
     end
 
     def size
-      node.all(:xpath, item_xpath).size
+      node.all(:xpath, item_xpath, options).size
```

Comment:
I've created a separate PR for the performance improvement https://github.com/appfolio/ae_page_objects/pull/191